input_reader.py

```diff
@@ -1,11 +1,103 @@
-class EncoderReader:
-    # TODO: enkoder nie?
-    def __init__(self) -> None:
-        pass
+from config import *
+import time
 
+DEBOUNCING_PERIOD_NS = 100_000
 
-class ButtonReader:
-    # TODO: przycisk tego ten
-    def __init__(self, button_color: str) -> None:
-        if button_color not in ["green", "red"]:
-            raise ValueError("button_color must be 'green' or 'red'")
\ No newline at end of file
+
+class Button:
```

Comment:
Przepisane 1:1 z C++, skoro tam działało to tu też powinno.

Reply:
🐍

Reply:
Ostatnie słowa przed tragedią 🙏